<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlFrame</class>
 <widget class="QFrame" name="ControlFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>64</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="mLabelCption">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Caption</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mLabelValue">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mButtonDec">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="mSpinDistance">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>32</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="mSpinColor">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>32</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mButtonInc">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
